src/set_theory/game/impartial.lean

```diff
@@ -0,0 +1,221 @@
+/-
+Copyright (c) 2020 Fox Thomson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Fox Thomson
+-/
+import set_theory.game.position
+import tactic.nth_rewrite.default
+import tactic
+
+universe u
+
+/-!
+# Basic deinitions about impartial (pre-)games
+
+We will define an impartial game, one in which left and right can make exactly the same moves.
+Our definition differs slightly by saying that the game is always equivilent to its negitve,
+no matter what moves are played. This allows for games such as poker-nim to be classifed as
+impartial.
+-/
+
+namespace pgame
+
+local infix ` ≈ ` := equiv
+
+/-- The definiton for a impartial game, defined using Conway induction -/
+@[class] def impartial : pgame → Prop
```

Comment:
Can you delete the line under `Future Work` in `pgame.lean`, now that you've done this?